discovery-provider/plugins/network-monitoring/docker-compose.yaml

```diff
@@ -26,40 +26,42 @@ services:
     networks:
       - audius_dev
   
-  grafana:
-    build:
-      context: grafana
-      args:
-        - GRAFANA_VERSION=latest
-        - GF_INSTALL_IMAGE_RENDERER_PLUGIN=true
-    user: 0:0
-    env_file:
-      - grafana/.env
-    links:
-      - prometheus:prometheus
-    ports: 
-      - "3000:3000"
-    restart: always
-    volumes:
-      - ./data/grafana:/var/lib/grafana
-    networks:
-      - audius_dev
+  # grafana:
```

Comment:
why comment these out?

Reply:
Because for now we're using the foundation's prometheus and grafana. It makes sense to uncomment them if you want to run a separate instance of prometheus and grafana but rn we don't

Reply:
got it, cool - can you leave comment indicating that, or instead of commenting these just change the docker-compose cmd to only bring up the other containers?